Library/Currency.swift

```diff
@@ -0,0 +1,72 @@
+import UIKit
+
+public enum Currency: Int {
```

Comment:
If you add `CaseIterable` protocol conformance to this enum you'll get `allCases` for free! 😃

Reply:
Awesome! 🎉